modules/overview-of-openshift-gitops.adoc

```diff
@@ -0,0 +1,39 @@
+[id="overview-of-openshift-gitops"]
+= Overview of OpenShift GitOps
+include::modules/common-attributes.adoc[]
+include::modules/gitops-document-attributes.adoc[]
+:context: overview-of-openshift-gitops
+toc::[]
```

Comment:
Please update the metadata information to match that for modules. Please refer to module templates in the [modular documentation guide](https://redhat-documentation.github.io/modular-docs/#introduction_{context}) to understand how modules are created. Or to a module in the modules folder of the OpenShift docs repository. This uses the template used for Assemblies in the OCP docs repository. The OpenShift specific guidelines are noted here, please ensure that you go through them: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc

Reply:
Removing the include statements and correcting the metadata in the modules might help the build succeed.